pkg/adaptor/server_test.go

```diff
@@ -206,13 +206,9 @@ func (p *mockProvider) CreateInstance(ctx context.Context, podName, sandboxID st
 		secondaryIP = "127.0.0.1"
 	}
 
-	ips := make([]net.IP, 2)
-	ips[0] = net.ParseIP(primaryIP)
-	ips[1] = net.ParseIP(secondaryIP)
-
-	if ips[0] == nil || ips[1] == nil {
-		return nil, fmt.Errorf("Could not parse IPs: primary IP %s and secondary IP %s", primaryIP, secondaryIP)
-	}
+	ips := make([]netip.Addr, 2)
+	ips[0] = netip.MustParseAddr(primaryIP)
```

Comment:
Which will panic on error? so that the err check was deleted?

Reply:
`netip.MustParseAddr(primaryIP)` will panic on errors.

https://pkg.go.dev/net/netip#MustParseAddr

I think this is OK in test code.